modules/reviewing-cluster-status-from-the-openshift-cluster-manager.adoc

```diff
@@ -9,14 +9,13 @@
 From the {product-title} web console, you can review detailed information about the status of your cluster on {cluster-manager}.
 
 .Prerequisites
-
+* You are logged in to {cluster-manager-url}.
 * You have access to the cluster as a user with the `cluster-admin` role.
 
 .Procedure
-
-. In the *Administrator* perspective, navigate to *Home* -> *Overview* -> *Details* -> *Cluster ID* -> *{cluster-manager}* to open your cluster's *Overview* tab in the {cluster-manager} web console.
-
-. From the *Overview* tab on {cluster-manager-url}, review the following information about your cluster:
+. Go to the *Clusters* list in {cluster-manager-url} and locate your {product-title} cluster.
```

Comment:
What about either: 
. Go to the *Clusters* list in {cluster-manager-url} and select your {product-title} cluster.

or 

. Go to the *Clusters* list in {cluster-manager-url} and locate your {product-title} cluster.
. Select the {product-title} cluster to open its details.
...

Reply:
Guess it is not applicable now.